api/write_test.go

```diff
@@ -84,23 +84,23 @@ func TestGzipWithFlushing(t *testing.T) {
 }
 func TestFlushInterval(t *testing.T) {
 	service := test.NewTestService(t, "http://localhost:8888")
-	writeAPI := NewWriteAPI("my-org", "my-bucket", service, write.DefaultOptions().SetBatchSize(10).SetFlushInterval(10))
+	writeAPI := NewWriteAPI("my-org", "my-bucket", service, write.DefaultOptions().SetBatchSize(10).SetFlushInterval(30))
 	points := test.GenPoints(5)
 	for _, p := range points {
 		writeAPI.WritePoint(p)
 	}
 	require.Len(t, service.Lines(), 0)
-	<-time.After(time.Millisecond * 15)
+	<-time.After(time.Millisecond * 50)
 	require.Len(t, service.Lines(), 5)
 	writeAPI.Close()
 
 	service.Close()
-	writeAPI = NewWriteAPI("my-org", "my-bucket", service, writeAPI.writeOptions.SetFlushInterval(50))
+	writeAPI = NewWriteAPI("my-org", "my-bucket", service, writeAPI.writeOptions.SetFlushInterval(500))
 	for _, p := range points {
 		writeAPI.WritePoint(p)
 	}
 	require.Len(t, service.Lines(), 0)
-	<-time.After(time.Millisecond * 60)
+	<-time.After(time.Millisecond * 550)
```

Comment:
I guess that we should get rid of any code that has to wait >500ms, maybe that the previous check is sufficient.

Reply:
The redundant long test was removed.